torch_geometric/datasets/md17.py

```diff
@@ -97,6 +97,115 @@ class MD17(InMemoryDataset):
             :obj:`torch_geometric.data.Data` object and returns a boolean
             value, indicating whether the data object should be included in the
             final dataset. (default: :obj:`None`)
+
+    Stats:
+        .. list-table::
+            :widths: 20 10 10 10 10 10
+            :header-rows: 1
+
+            * - Name
+              - #graphs
+              - #nodes
+              - #edges
+              - #features
+              - #classes
+            * - Benzene FHI-aims
+              - 49,863
+              - 12
+              - 0
+              - 0
+              - 0
+            * - Benzene
+              - 627,983
+              - 12
+              - 0
+              - 0
+              - 0              
+            * - Benzene CCSD-T
```

Comment:
Using `CCSD(T)` would result in a building error:
```bash
Handler <function setup.<locals>.rst_jinja_render at 0x1273eb160> for event 'source-read' threw an exception (exception: 'NoneType' object has no attribute 'group')
```
Had to use `CCSD-T` instead.